### ISSUE TITLE
Commas before closing brackets do not look consistent

### ISSUE THREAD

**Issue by dmitrytrager** [rubocop/rubocop#10971]
Rubocop has several style cops related to trailing commas. The meaning of trailing commas is often to allow adding new method arguments or structure elements without changing previous ones.

However, having syntax like [this](https://github.com/rubocop/rubocop/blob/8a80173b48bbd16224903f462093dd985374f62b/spec/rubocop/cop/style/trailing_comma_in_arguments_spec.rb#L443) I won't be able to add new argument at the end without changing lines with existing arguments.

Should we disable comma before closing brackets?

--------

## Expected behavior

Commas right before closing brackets should not be allowed

## Actual behavior
```
some_method(a: "b",
  c: "d",)
```
is a valid syntax

## Steps to reproduce the problem

1. Use this cop
```
Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: consistent_comma
```
2. Add code described above
3. Run rubocop

## RuboCop version

```
$ bin/rubocop -V
1.31.2 (using Parser 3.1.2.0, rubocop-ast 1.18.0, running on ruby 3.1.2 x86_64-darwin21)
  - rubocop-performance 1.11.5
  - rubocop-rails 2.12.4
  - rubocop-rspec 2.4.0
```

**Comment by ngouy** [comment]
Shopify-rubocop just kicked in with that cop, and I do agree that

```rb
call(
  a,
  b,
  ) # is ok, last comma should be enfored
```

But that
```rb
call(a,
       b,) # enforcing that here is weird, and anyway useless because even with a `,` it's still a 2 line diff

call(a,
       b, # diff line 1
       c,) # diff line 2
```
